modules/administration-guide/partials/ref_grafana-dashboards-for-the-dev-workspace-operator.adoc

```diff
@@ -1,39 +1,36 @@
 [id="ref_grafana-dashboards-for-the-dev-workspace-operator"]
-= Grafana dashboard for the {devworkspace} Operator
+= Dashboard for the {devworkspace} Operator
 
-The example Grafana dashboard link:https://github.com/devfile/devworkspace-operator/blob/main/docs/grafana/grafana-dashboard.json[based on `grafana-dashboard.json`] displays the following metrics from the {devworkspace} Operator.
+The OpenShift web console custom dashboard is based on Grafana 6.x and displays the following metrics from the {devworkspace} Operator.
 
-== The *{devworkspace}-specific metrics* panel
+NOTE: Not all features for Grafana 6.x dashboards are supported as an OpenShift web console dashboard.
+
+== {devworkspace} metrics
+
+The {devworkspace}-specific metrics are displayed in the *{devworkspace} Metrics* panel.
+
+.The {devworkspace} Metrics panel
 
 .The *{devworkspace}-specific metrics* panel
```

Comment:
IMO, line `12` above has the correct text and needs to stay, while these two lines needs to be deleted:
```suggestion
```